program/src/processor.rs

```diff
@@ -2982,8 +3024,21 @@ impl Processor {
 
         // TODO OPT make this more efficient
         msg!(
-            "liquidate_perp_market details: {{ \"market_index\": {}, \"base_transfer\": {}, \"quote_transfer\": {}, \"bankruptcy\": {} }}",
+            "liquidate_perp_market details: {{ \
```

Comment:
We have to make sure the compute used here is not too bad. @riordanp Can you test this out?

Reply:
Just tried this on devnet, the extra logging costs 35670 compute

Reply:
Additional 35k or 35k in total? As long as it clears with a lot of extra compute leftover I think we're safe